ledger/acctonline.go

```diff
@@ -556,6 +556,26 @@ func (ao *onlineAccounts) onlineCirculation(rnd basics.Round, voteRnd basics.Rou
 	return totalStake, nil
 }
 
+// roundsParamsEx return the round params for the given round for extended rounds range
+// by looking into DB as needed
+// locking semantics: requires accountsMu.RLock()
+func (ao *onlineAccounts) roundsParamsEx(rnd basics.Round) (ledgercore.OnlineRoundParamsData, error) {
+	paramsOffset, err := ao.roundParamsOffset(rnd)
```

Comment:
Why not extend to the db lookup inside `roundParamsOffset`? Are there cases where we rather return an error than look into the db?

Reply:
because `roundParamsOffset` returns offseet, not data